sdk/cosmosdb/Azure.ResourceManager.CosmosDB/tests/ScenarioTests/GraphResourcesOperationsTests.cs

```diff
@@ -0,0 +1,162 @@
+            ThroughputSettingsGetResults throughputSettingsGetResults = throughputResponse.ConfigureAwait(false).GetAwaiter().GetResult();
+            Assert.NotNull(throughputSettingsGetResults);
+            Assert.NotNull(throughputSettingsGetResults.Name);
+            Assert.AreEqual(throughputSettingsGetResults.Resource.Throughput, sampleThroughput);
+            Assert.AreEqual(graphThroughputType, throughputSettingsGetResults.Type);
+
+            IndexingPolicy indexingPolicy = new IndexingPolicy(true, IndexingMode.Consistent, new List<IncludedPath> { new IncludedPath { Path = "/*" } }, new List<ExcludedPath> { new ExcludedPath { Path = "/pathToNotIndex/*" } }, new List<IList<CompositePath>>
+                            {
+                                new List<CompositePath>
+                                {
+                                    new CompositePath { Path = "/orderByPath1", Order = CompositePathSortOrder.Ascending },
+                                    new CompositePath { Path = "/orderByPath2", Order = CompositePathSortOrder.Descending }
+                                },
+                                new List<CompositePath>
+                                {
+                                    new CompositePath { Path = "/orderByPath3", Order = CompositePathSortOrder.Ascending },
+                                    new CompositePath { Path = "/orderByPath4", Order = CompositePathSortOrder.Descending }
+                                }
+                            }, new List<SpatialSpec> { new SpatialSpec ( "/*", new List<SpatialType> { new SpatialType("Point") } ) });
+
+            ContainerPartitionKey containerPartitionKey = new ContainerPartitionKey(new List<string> { "/address" }, "Hash", null);
+            IList<string> paths = new List<string>() { "/testpath" };
+            UniqueKey uk = new UniqueKey(paths);
+            IList<UniqueKey> uniqueKeys = new List<UniqueKey>();
+            uniqueKeys.Add(uk);
+            UniqueKeyPolicy uniqueKeyPolicy = new UniqueKeyPolicy(uniqueKeys);
+
+            ConflictResolutionPolicy conflictResolutionPolicy = new ConflictResolutionPolicy(new ConflictResolutionMode("LastWriterWins"), "/path", "");
+
+            GremlinGraphCreateUpdateParameters gremlinGraphCreateUpdateParameters = new GremlinGraphCreateUpdateParameters(new GremlinGraphResource(gremlinGraphName, indexingPolicy, containerPartitionKey, -1, uniqueKeyPolicy, conflictResolutionPolicy), new CreateUpdateOptions(sampleThroughput, default));
+
+            Response<GremlinGraphGetResults> gremlinResponse = await WaitForCompletionAsync(await CosmosDBManagementClient.GremlinResources.StartCreateUpdateGremlinGraphAsync(resourceGroupName, databaseAccountName, databaseName, gremlinGraphName, gremlinGraphCreateUpdateParameters));
+            GremlinGraphGetResults gremlinGraphGetResults = gremlinResponse.Value;
+            Assert.NotNull(gremlinGraphGetResults);
```

Comment:
Assert the response is not null before assert the value of the response is not null

Reply:
Resolved